x/ccv/consumer/keeper/keeper_test.go

```diff
@@ -17,19 +18,19 @@ import (
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 )
 
-// TestUnbondingTime tests getter and setter functionality for the unbonding period of a consumer chain
+// TestUnbondingTime tests the getter and setter for the unbonding period of a consumer chain,
 func TestUnbondingTime(t *testing.T) {
 
 	consumerKeeper, ctx, ctrl, _ := testkeeper.GetConsumerKeeperAndCtx(t, testkeeper.NewInMemKeeperParams(t))
 	defer ctrl.Finish()
+	consumerKeeper.SetParams(ctx, consumertypes.DefaultParams())
 
-	_, ok := consumerKeeper.GetUnbondingTime(ctx)
-	require.False(t, ok)
-	unbondingPeriod := time.Hour * 24 * 7 * 3
-	consumerKeeper.SetUnbondingTime(ctx, unbondingPeriod)
-	storedUnbondingPeriod, ok := consumerKeeper.GetUnbondingTime(ctx)
-	require.True(t, ok)
-	require.Equal(t, storedUnbondingPeriod, unbondingPeriod)
+	ubdTime := consumerKeeper.GetUnbondingPeriod(ctx)
+	require.Equal(t, consumertypes.DefaultConsumerUnbondingPeriod, ubdTime)
+
+	consumerKeeper.SetUnbondingPeriod(ctx, time.Hour*24*10)
+	storedUnbondingPeriod := consumerKeeper.GetUnbondingPeriod(ctx)
+	require.Equal(t, time.Hour*24*10, storedUnbondingPeriod)
 }
```

Comment:
I think we no longer need this test. This was needed for testing `SetUnbondingTime` and `GetUnbondingTime`, which were removed (moved to `param.go`). The testing should be part of `params_test.go`, see `TestParams`.

Reply:
Nice catch 👍